feature_engine/transformation/date_transformer.py

```diff
@@ -0,0 +1,49 @@
+from typing import List, Optional, Union
+
+import numpy as np
+import pandas as pd
+
+from feature_engine.base_transformers import BaseNumericalTransformer
+from feature_engine.variable_manipulation import _check_input_parameter_variables
+
+
+class date_feature_extractor(BaseNumericalTransformer):
```

Comment:
would it be better to call the class `DatetimeFeatureExtractor` ?

Reply:
Also, could we place the script in a separate folder called datetime? not within the folder transformers?